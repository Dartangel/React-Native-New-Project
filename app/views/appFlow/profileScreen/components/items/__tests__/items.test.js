import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Items from '../index';

describe('Items', () => {
    test('Items snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Items />);
        expect(snapshot).toMatchSnapshot();
    });
});