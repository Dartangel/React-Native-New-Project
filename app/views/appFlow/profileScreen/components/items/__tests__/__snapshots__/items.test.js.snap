// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Items Items snapshot 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#d2d2d2",
      "borderBottomWidth": 2,
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "flex": 0.8,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 0.2,
          "justifyContent": "center",
        }
      }
    >
      <Icon
        allowFontScaling={false}
        color="#d2d2d2"
        size={30}
      />
    </View>
    <View
      style={
        Object {
          "flex": 0.8,
          "justifyContent": "center",
        }
      }
    >
      <TouchableOpacity
        activeOpacity={0.2}
        delayLongPress={600}
        extraButtonProps={
          Object {
            "rippleColor": "transparent",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#ffffff",
              "fontSize": 20,
              "paddingLeft": 10,
            }
          }
        />
      </TouchableOpacity>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 0.2,
        "flexDirection": "row",
        "justifyContent": "space-around",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#08DBA5",
          "fontSize": 15,
        }
      }
    />
  </View>
</View>
`;
